packages/renderer/src/lib/actions/ActionUtils.spec.ts

```diff
@@ -0,0 +1,72 @@
+import '@testing-library/jest-dom/vitest';
```

Comment:
FYI: 
1. we should not have UI imports on this part
1. the file header with copyright is missing